src/__tests__/Carousel.tsx

```diff
@@ -784,6 +784,36 @@ describe('Slider', function() {
         });
     });
 
+    describe('Focus', () => {
+        describe('calling forceFocus', () => {
+            it('should call carousel wrapper focus', () => {
+                componentInstance.carouselWrapperRef.focus = jest.fn();
+                componentInstance.forceFocus();
+                expect(componentInstance.carouselWrapperRef.focus).toHaveBeenCalledTimes(1);
+            });
+        });
+
+        describe('AutoFocus === true', () => {
```

Comment:
@leandrowd I couldn't tell how you were testing if methods were being invoked inside of componentDidMount based on initial props, so I introduced jest.spy. If this is not desirable, I can remove it.